src/digitalIdentity-frontend/src/app/services/backend-http-service/backend-http-service.service.ts

```diff
@@ -29,7 +29,7 @@ export class BackendHttpService {
           params: params,
         })
         .pipe(
-          timeout(20000),
+          timeout(60000),
```

Comment:
Why is this time changed to 60s now?

Reply:
Because you said that you have a timeout when doing your task and said something about 30 seconds. But it doesn't really matter. I've reverted it to 20